features/portfolio_search.feature

```diff
@@ -11,3 +11,11 @@ Scénario: une valeur d'attribut
   Alors l'item "AXN 009" est affiché
   Mais l'item "SJ 001" est caché
 
+Scénario: search for a creator
```

Comment:
Profitez en, dans les scénarios ont peut utiliser la langue française 😄 L'idée est de les rendre le plus compréhensibles possible pour les parties prenantes.

Reply:
Ce n'est pas très grave, mais, par ailleurs, en mettant ici un nom de scénario  avec un verbe, vous "cassez" un peu l'homogénéité de l'affichage "summary" de la fonctionnalité par cucumber : 

```
Sélectionner des items en cherchant
- une valeur d'attribut
- search for a creator
```